server/controllers/finance/priceList.js

```diff
@@ -36,9 +36,16 @@ function lookup(req) {
 
   if (req.query.detailed === '1') {
     sql = `
-      SELECT BUID(pl.uuid) as uuid, pl.label, pl.created_at, pl.description, COUNT(pl_it.uuid) as itemsNumber
+      SELECT BUID(pl.uuid) as uuid, pl.label, pl.created_at, pl.description,
+        COUNT(pl_it.uuid) as itemsNumber, groups.nbr as subcribedGroupsNumber
       FROM price_list pl
       LEFT JOIN price_list_item pl_it ON pl_it.price_list_uuid = pl.uuid
+      LEFT JOIN (
```

Comment:
Why does this need to be a subquery?  Can't we do this in the same format as the `itemsNumber`?

Reply:
There is an important ```GROUP BY``` in the subquery

Reply:
Try
```sql
LEFT JOIN patient_group ON patient_group.price_list_uuid = pl.uuid
```

And then put `COUNT(patient_group.uuid)  as subcribedGroupsNumber` in the SELECT part.  I think ti should behave the same, but save quite a bit of work.  JOINs are typically more standard than writing subqueries.